modules/apps/upload/upload-web/src/main/java/com/liferay/upload/web/internal/HTMLImageAttachmentElementHandler.java

```diff
@@ -54,17 +56,27 @@ public String replaceAttachmentElements(
 
 		StringBuffer sb = new StringBuffer(content.length());
 
+		Map<String, FileEntry> fileEntries = new HashMap<>();
+
 		while (matcher.find()) {
-			FileEntry tempAttachmentFileEntry = _getFileEntry(matcher);
+			String match = matcher.group(0);
+
+			FileEntry attachmentFileEntry = fileEntries.get(match);
```

Comment:
Can you rename `attachmentFileEntry` to simply `fileEntry`?

Reply:
See SF commit